docs/docs/references/using-cli/custom-ui-local-proxy.mdx

```diff
@@ -0,0 +1,161 @@
+
+```mermaid
+sequenceDiagram
+    box Local machine
+    participant A as Your application
+    end
+    box Logto Cloud
+    participant B as Logto Cloud auth endpoint
+    participant C as Logto sign-in page
+    end
+    A ->> B: User initiates "sign-in" action and request auth
+    B -->> A: Return auth response and tell the client<br/>to redirect to the Logto sign-in page
+    A ->> C: Redirect to the Logto sign-in page
+    C ->> B: Submit the sign-in form and<br/>request experience API to authenticate
+    B -->> C: Respond the sign-in request and<br/>tell the client to redirect to your application
+    C -->> A: Redirect to your application
+    A --> A: Handle the sign-in callback and<br/>the user is now authenticated
+```
+
+But now since you are developing your own custom sign-in UI, you need a way to navigate to the custom sign-in UI pages running on your local machine instead.
+This requires a local proxy to intercept the outgoing requests from your application and redirect them to your custom sign-in UI pages.
+
+Additionally, you need to interact with Logto's experience API to authenticate users and manage sessions.
+This proxy will also help forward these experience API requests to Logto Cloud in order to avoid CORS issues.
+
+The sequence diagram below illustrates how a successful "sign-in" flow works with your custom UI and the proxy in place:
+
+```mermaid
+sequenceDiagram
+    box Local machine
+    participant A as Your application
+    participant B as Your custom sign-in UI
+    participant C as proxy
+    end
+    box Logto Cloud
+    participant D as Logto Cloud auth endpoint
+    participant E as Logto sign-in page
+    end
+    A ->> C: User initiates "sign-in" action and request auth
+    C ->> D: Forward auth request to Logto Cloud endpoint
+    D -->> C: Return auth response andtell the client<br/>to redirect to the Logto sign-in page
+    C ->> B: Intercept the redirect and<br/>redirect to your custom sign-in page
+    B ->> C: Submit the sign-in form and<br/>request experience API to authenticate
+    C ->> D: Forward the experience API requests to Logto Cloud
+    D -->> C: Authenticate sign-in request and<br/>tell the client to redirect to your application
+    C -->> A: Redirect to your application
+    A --> A: Handle the sign-in callback and<br/>the user is now authenticated
+```
+
+With the proxy in place, you can now develop and test your custom sign-in UI locally, without needing to upload the assets to Logto Cloud every time you make a change.
+
+## Instructions
+
+### Step 1: Execute the command
+
+Assuming your Cloud tenant ID is `foobar`, and you have a custom sign-in page running on your local dev server at `http://localhost:4000`.
+
+Then you can execute the command this way:
+
+<Tabs groupId="cmd">
+
+  <TabItem value="cli" label="CLI">
+
+```bash
+logto proxy -p 9000 --experience-uri http://localhost:4000/ --endpoint https://foobar.logto.app/
```

Comment:
this command should be updated accordingly?

Reply:
Do you mean we should call this command `logto tunnel` instead of `logto proxy`?